scripts/update/02-run.sh

```diff
@@ -12,4 +12,13 @@ echo "========= Stage: Post-update =========="
 echo "======================================="
 echo
 
-# Nothing here for now
+# Make Umbrel OS specific post-update changes
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
+
+  # Remove dphys-swapfile since we now use our own swapfile logic
+  if command -v dphys-swapfile >/dev/null 2>&1; then
```

Comment:
Do you think we can move this logic to `01-run.sh` instead? Right after this block of code:

https://github.com/getumbrel/umbrel/blob/8c5bf7aabd71450b1ebba6c959bac85a7e13a34d/scripts/update/01-run.sh#L32

Reply:
It can definitely do there, I just thought it belonged in post-update.

Why do you think removing an old dependency should be part of the install stage instead of the post-update stage?

And what kind of logic do you imagine should be in the post-update stage?

Reply:
Yeah tbh, it's objective I guess. I think this specific change of uninstalling a dependency can go alongside the manual creation and enabling of the swap file, which might be better suited at the install stage. Also, it may help to keep all Umbrel OS modifications in the same script/block so they're easier to just remove when we release an update that pins another release.

Eventually, we can end up dedicating the entire `01-run.sh` for Umbrel OS installation changes, and let it skip directly to `02-run.sh` for non-Umbrel OS systems. 

As for the post-update stage, that could be better suited for unit checks to make sure everything has been installed successfully.

Reply:
Also, it could be a good idea to import your semver bash library or just wrap the swap logic inside:

if [[ $UMBREL_OS == "v0.1.2" ]] || [[ $UMBREL_OS == "v0.2.3" ]] {
  remove dphys-swapfile
  create a 4GB swap and enable it
}

Reply:
I'm not sure we should limit with semver.

Checking that we are on Umbrel OS and if `dphys-swapfile` exists more accurately conveys what we're trying to do.

If for some reason they still had it installed and skipped an update, or we were to not accurately describe the version range, or mess up a future update version requirement, or they are on a commit version, or we're testing locally on a dev branch, are all scenarios where that could fail unpredictably.

Just saying "if we're on Umbrel OS and if this thing exists, uninstall it" is simpler, less error prone and more accurately describes what we are doing.